lib/galaxy/webapps/galaxy/api/workflows.py

```diff
@@ -82,6 +82,10 @@ def set_workflow_menu( self, trans, **kwd ):
             workflow_ids = []
         elif type( workflow_ids ) != list:
             workflow_ids = [ workflow_ids ]
+        workflow_ids_decoded = []
```

Comment:
I'd prefer the compacter `workflow_ids = [trans.security.decode_id(wf_id) for wf_id in workflow_ids]`, but that's not a must. I do prefer not to use the plural (`ids`) in the below for loop though.